core/src/main/java/org/owasp/dependencycheck/data/nodeaudit/NodeAuditSearch.java

```diff
@@ -127,6 +142,14 @@ public NodeAuditSearch(Settings settings) throws MalformedURLException {
                         final NpmAuditParser parser = new NpmAuditParser();
                         return parser.parse(jsonResponse);
                     }
+                case 503:
+                    LOGGER.debug("Invalid payload submitted to Node Audit API. Received response code: {} {}",
```

Comment:
A 503 is not due to an invalid payload?

```suggestion
                    LOGGER.debug("Node Audit API returned `{} {}` - retrying request.",
```

Reply:
Yeah sorry copy and paste mistake ..